server.js

```diff
@@ -95,5 +96,9 @@ app.post('/unwrap', function(req, res) {
 app.get('/');
 
 app.get('*', function (req, res) {
-    res.render(path.join(__dirname, '/index.html'),{defaultUrl: VAULT_URL_DEFAULT, defaultAuth: VAULT_AUTH_DEFAULT});
+    res.render(path.join(__dirname, '/index.html'),{
+        defaultUrl: VAULT_URL_DEFAULT,
+        defaultAuth: VAULT_AUTH_DEFAULT,
+        suppliedAuthToken: VAULT_SUPPLIED_TOKEN_HEADER ? req.header(VAULT_SUPPLIED_TOKEN_HEADER) : ""
```

Comment:
No need to set it to a falsy value, req.header() returns undefined if not found which is falsy anyways.